src/tests/ftest/ior/intercept_multi_client.yaml

```
@@ -29,34 +29,33 @@ ior:
     client_processes:
       np: 16
     test_file: testFile
-    repetitions: 1
-# Remove the below line once DAOS-3143 is resolved
-    dfs_destroy: False
+    repetitions: 3
     iorflags:
         ssf:
-          flags: "-k -v -D 300 -w -r"
+          flags: "-k -v -D 600 -w -r"
```

1. Since we're relying on stonewall to limit IO time, we could simplify the `block_size` specification to something like this:
```suggestion
          flags: "-k -v -D 600 -w -r"
          block_size: 100G
```
And remove `block_size` from each section below. That would cause each `!mux` variant to run for `-D 600`s, realistically never actually writing `100G`.
2. There are 4 `!mux` variants and `-D 600` is 10 minutes. Which means `10m * 2 * 4 = 80m` for these tests. Do we really need 600s of stonewall? FWIW, IO500 uses 300s as a minimum.

---

We're not, if we hit the stonewall then different tests will send different amounts of data so you can't compare them, so we increase the stonewall so it shouldn't occur, and set the fail_on_warning flag to fail the test if stonewall occurs.  Effectively it's a safety mechanism at this point.